### ISSUE TITLE
[Feature request] dark theme for desktop apps

### ISSUE THREAD

**Issue by tessus** [laurent22/joplin#400]
### Operating system

- macOS
- Linux

### Application

- Desktop

---

Sometimes the light theme can be a strain on the eyes.
For some reason the dark theme only seems to be available for the mobile apps.

Is there any chance this dark theme could be added to the desktop applications as well?

**Comment by laurent22** [comment]
Duplicate of #93

**Comment by laurent22** [comment]
In general, for all theme-related requests, the issue is the desktop app currently supports only one theme. Once support for a second one is done, it will be possible to add any kind of theme.

**Comment by tessus** [comment]
Ok, thanks for the info. I saw #93, but it looked like a mobile issue, thus not related to the desktop, especially since mobile versions already have more than 1 theme.

**Comment by Munchotaur** [comment]
This feature would be a huge bonus for users with eyesight that struggles with light themes. Accessibility should always be fairly high up the list of priorities!
